src/reconcile/rec_visibility.c

```diff
@@ -160,7 +163,14 @@ __rec_append_orig_value(
 
             WT_ERR(__wt_upd_alloc_tombstone(session, &tombstone, &size));
             total_size += size;
-            tombstone->txnid = unpack->tw.stop_txn;
+            /*
+             * When evicting a page during recovery, we need to clear the transaction id as it is
```

Comment:
I feel this comment needs more details. It is not only evicting during recovery, recovery checkpoint also can append the on-page value. `when reconciling during recovery`?

Also, mention the reason behind the transaction ID reset. WT restarts the transaction ID numbers upon restart, we reset reading any transaction IDs stored on-disk after recovery, but we must reset the transaction ID of updates that are created during recovery to avoid using the previous run transaction ID.

Reply:
@quchenhao, I think your last update misses what @kommiharibabu is suggesting here:
```
we must reset the transaction ID of updates that are created during recovery to avoid using the previous run transaction ID
```

Reply:
In this case, the txnid is not created during recovery but from the previous run.